### PR TITLE
Update da.json

### PR DESCRIPTION
**Pull request by iplayer1337fivem**
## Checklist

- [x ] I have personally loaded this code into an updated Qbox project and checked all of its functionality.
- [x ] My pull request fits the contribution guidelines & code conventions.

### CODE REVIEW

File: locales/da.json [hunk 1]
```diff
@@ -6,46 +6,47 @@
     "addkeys_id_help": "Spiller ID",
     "addkeys_plate": "nummerplade",
     "addkeys_plate_help": "Nummerplade",
-    "givekeys": "givede nøgler",
-    "givekeys_help": "Overlever nøglerne til nogen. Hvis der ikke er nogen ID, gives de til den nærmeste person eller alle i køretøjet.",
+    "givekeys": "givnøgler",
```
Comment (ArturMichalak) [inline locales/da.json:9]: How about using “o” instead of “ø”?

**Review by ArturMichalak — APPROVED** [review]

File: locales/da.json (hunk 1 above)
Comment (iplayer1337fivem) [reply locales/da.json:9]: its a danish letter, pronounced "oeeh"

Comment (ArturMichalak) [reply locales/da.json:9]: I suggested this to simplify the command writing

Comment (iplayer1337fivem) [reply locales/da.json:9]: well, we could make a lookalike with a capital Q, but its a mess. So ty for letting Ø, Æ and Å into the mix <3

**Pull request merged**